inc/saiacl.h

```diff
@@ -3204,10 +3208,19 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_ACTION_CHAIN_REDIRECT = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x38,
 
+    /**
+     * @brief Disable ARS forwarding for a flow
+     *
+     * @type sai_acl_action_data_t bool
+     * @flags CREATE_AND_SET
+     * @default disabled
+     */
+    SAI_ACL_ENTRY_ATTR_ACTION_DISABLE_ARS = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x39,
```

Comment:
add comment that this attribute for ACL is applicable only if the global ARS is enabled.